Posts/2021/03/ScriptingGuy.0007.folderexists.md

```diff
@@ -0,0 +1,41 @@
+---
+post_title: Determine if a folder exists
+username: tfl@psp.co.uk
+Catagories: PowerShell
+tags: File, Test-Path
+Summary: How can I determine if a folder exists?
+---
+
+**Q:** Is there any way to determine whether or not a specific folder exists on a computer?
+**A:**  There are loads of ways you can do this.
+A simple way is to use the power of array handling in PowerShell.
```

Comment:
??

Reply:
The easiest way to to this is to use the `Test-Path` cmdlet.

Reply:
resolved.